@here/harp-mapview/lib/MapView.ts

```diff
@@ -2528,7 +2548,7 @@ export class MapView extends THREE.EventDispatcher {
         const theme = this.m_theme;
         if (this.m_skyBackground instanceof SkyBackground && theme.sky !== undefined) {
             // there is a sky in the view and there is a sky option in the theme. Update the colors
-            this.updateSkyBackgroundColors(theme.sky, theme.clearColor);
+            this.updateSkyBackgroundColors(theme.sky, theme.clearColor, this.projection.type);
```

Comment:
3rd parameter is not needed here as it accessible in `updateSkyBackgroundColors`

Reply:
opps, done.